resources/views/top/index.blade.php

```diff
@@ -9,10 +9,14 @@
       <a href="{{ url('/') }}">TOP</a>
     <form method="post" action="{{ url('/notifications') }}"  enctype="multipart/form-data"> 
     @csrf
-    <div class="badge">  
+    @if(!$notifications == null) 
```

Comment:
この様に条件を否定する場合は `==` ではなく `!=` を使いましょう
```suggestion
    @if($notifications != null) 
```